### ISSUE TITLE
[Feature request] Print tranq rotation in chat

### ISSUE THREAD

**Issue by lantisnt** [Slivo-fr/TranqRotate#16]
Please reformat the display of rotation to:
1. ...
2. ...
3. ...
...
Backups: ..., ..., ...

To make it more readable.

Please also make the tranq display a slashcommand also instead of button only:
```lua
	elseif (cmd == 'broadcast') then
		TranqRotate:broadcastToRaid()
```

**Comment by lantisnt** [comment]
NVM. Im blind.

**Comment by lantisnt** [comment]
Example of how i did it locally just for our todays raid:
```lua
-- Sends rotation setup to raid channel
function TranqRotate:broadcastToRaid()
    local channel = 'RAID'

    if (IsInRaid()) then

        SendChatMessage('--- ' .. L['BROADCAST_HEADER_TEXT'] .. ' ---', channel)
        TranqRotate:displayRotationOneByOne(TranqRotate.rotationTables.rotation, channel)

        if (#TranqRotate.rotationTables.backup > 0) then
            SendChatMessage(
                TranqRotate:buildGroupMessage(L['BROADCAST_BACKUP_PREFIX'] .. ' : ', TranqRotate.rotationTables.backup),
                channel
            )
        end
    end

end

-- Display all hunters one bye one
function TranqRotate:displayRotationOneByOne(rotationTable, channel)
    local order = 1;
    for key, hunt in pairs(rotationTable) do
        SendChatMessage(tostring(order)..". "..hunt.name, channel)
        order = order + 1;
    end
end
```
![tranq_rotate_readable](https://user-images.githubusercontent.com/43088589/76166999-a3717600-6163-11ea-9637-a44452841e15.png)

**Comment by Slivo-fr** [comment]
Hey,

Thanks for contributing,

I'm unsure about this issue : This is indeed a bit more readable but at the cost of 2-3 lines of text in raid chat. That could be a bit spammy. I need to think about it and ask some more opinions, maybe I could add this as an option later ?

**Comment by Slivo-fr** [comment]
Added, will be shipped with the 1.4 release :)

**Comment by Slivo-fr** [comment]
Added in v1.4.0 ! Please report any issue !
